README.md

```diff
@@ -0,0 +1,70 @@
+# Vanilla REST api
+
+This vanilla API allows a user to make GET, POST, and DELETE requests to the Artist object. The user can POST a new Artist name and genre, receive an id, and can delete the Artist by that id.
+
+### Set-Up
+
+In your Terminal
+
+```sh
+$ cd 08-vanilla_rest_api
+$ npm i
+```
+This will install the proper dependencies. You should receive the following in your package.json file:
+
+```sh
+"dependencies": {
+    "bluebird": "^3.4.6",
+    "node-uuid": "^1.4.7",
+    "chai": "^3.5.0",
+    "mocha": "^3.2.0",
+    "superagent": "^3.3.0"
+  }
+```
```

Comment:
Minor nitpick- it's not necessary to include dependencies in a README since they are accessible in your package.json :)